README.md

```diff
@@ -1 +1,77 @@
-# react-group-project
\ No newline at end of file
+# Space Travelers' Hub
+
+In this task, we worked with the real live data from the SpaceX API. Our task was to build a web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+
```

Comment:
- [ ] Isn't going to be better if you attached an image or two of your great app in the README file to be marked as a 100% README file? 😉 💯